operator/cmd/mesh/install.go

```diff
@@ -236,9 +240,99 @@ func InstallManifests(iop *v1alpha12.IstioOperator, force bool, dryRun bool, res
 		return iop, err
 	}
 
+	if dryRun {
+		currentObj, err := reconciler.GetObject(iop)
+		if err != nil {
+			return iop, err
+		}
+		obj, err := object.ParseYAMLToK8sObject([]byte(iopStr))
+		if err != nil {
+			return iop, err
+		}
+		expectedObj := obj.UnstructuredObject()
+		result, err := showDifferences(currentObj, expectedObj)
+		if err != nil {
+			return iop, err
+		}
+		fmt.Println(result)
+	}
 	return iop, saveIOPToCluster(reconciler, iopStr)
 }
 
+func showDifferences(existing, target *unstructured.Unstructured) (string, error) {
+	// We are only interested in seeing the differences between the specs
+	obj := make(map[string]interface{})
+	obj["spec"] = existing.Object["spec"]
+	existing.Object = obj
```

Comment:
It seems a bit confusing to modify the `existing` object. I think thisi s trying to get the spec yaml, right? Can we just do yaml.marshal(existing.Object["spec"])?

Reply:
@howardjohn I can try that and see if both produces same thing. Thanks for quickly reviewing this PR.

Reply:
@howardjohn no longer need to do that. see the latest commit. Thanks.